service/src/test/scala/com/yahoo/maha/service/calcite/DefaultMahaCalciteSqlParserTest.scala

```diff
@@ -336,4 +336,23 @@ class DefaultMahaCalciteSqlParserTest extends BaseMahaServiceTest with Matchers
 
     assert(request.filterExpressions.toString contains "EqualityFilter(Student ID,123,false,false)")
   }
+
+  test("test filter: not equals") {
+
+    val sql =
+      s"""
+              select * from student_performance
+              where 'Student ID' != 123
```

Comment:
can you add test for not equals with string

Reply:
Done